### DISCUSSION TITLE
What set urls in lowercase and title not containing the site name?

### DISCUSSION THREAD

**Discussion by ooker777** [squidfunk/mkdocs-material#6206] [Q&A] [answered]
I catch in Google Analytics that some entries are duplicated and some don't:

![](https://i.imgur.com/vH34kDM.png)

For duplicated entries, one has site name in its and one doesn't. Further investigation shows that for titles having site name in them, their URLs have correct case (e.g. ` /👏Trấn Kỳ/Hướng dẫn sử dụng Trấn Kỳ/`), while for titles not having site name in them their URLs are lowercased (e.g. `/👏trấn kỳ/hướng dẫn sử dụng trấn kỳ/`).

![](https://i.imgur.com/RKkorRsm.png)

What make them lowercased? Why does that make the title not containing the site name? Why don't they lowercase all at once?

**Comment by alexvoss** [comment]
It sounds to me like this might be an issue with MkDocs (if it is indeed an issue) rather than Material for MkDocs. At least this is what the [section on setting page titles](https://squidfunk.github.io/mkdocs-material/reference#setting-the-page-title) suggests to me.

It might help to produce a [minimum reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/?h=reproduction#creating-a-reproduction) that shows the problem and allows people to investigate it. One question I would ask is whether this happens with other languages as well as Vietnamese? I will go and have a quick look and try with English and German.

**Reply by alexvoss** [reply]
So, I created a page `Trấn Kỳ/Hướng dẫn sử dụng Trấn Kỳ.md` in a test project and the URL and page title come out in uppercase. I am not entirely sure I fully understand the problem. In the screenshot from Google Analytics, the field listed is the page title and the issue there does not seem to be capitalization but that the titles highlighted are actually different - one version seems to have additional text appended. In your description you refer to the URLs, which are a different issue. A reproduction with a clearer description of what the issue is would be helpful - to us or to the developers of MkDocs should this be an issue for them.

**Reply by ooker777** [reply]
sorry, I forgot to link the repo: https://github.com/QuaCau-TheSphere/Cw-xu-ly-du-lieu-va-lap-trinh. I'll try to isolate the bug tomorrow. I was trying to understand Mkdocs in general

**Reply by ooker777** [reply]
hmm, somehow in the meta.html there is a line `<script> document.title = "{{ title }}" </script>` and I don't know how it got there. I have deleted it and let's see hot it goes. In the meantime, does MkDocs automatically redirects all same URLs with different cases to the same file? If yes, which code does that?

**Comment by alexvoss** [comment]
Hm, I am relatively new to this - what is `meta.html`?

I doubt that MkDocs does any such thing as redirecting as you describe. If you find that you can access pages with URLs having different capitalization then it is more likely to be the server - or the filesystem of the server - that makes things case-insensitive.

**Reply by ooker777** [reply, marked as answer]
It's in `overrides/partials`. Actually I'm working with [Obsidian Pulisher](https://obsidian-publisher.netlify.app/), a child theme of MM. Since I'm new to both Mkdocs and MM, so it's possible that I asked in a wrong forum. 

I use Netlify to deploy the website. I think it makes URLs case-insensitive. However I still not sure what should I do to investigate the problem better.

**Reply by alexvoss** [reply]
Had another look and I cannot find `meta.html` in Material for MkDocs, so I assume it is part of Obsidian Publisher. Best I can do for now is to suggest you look at their [discussions](https://github.com/orgs/ObsidianPublisher/discussions) to see if someone else had the same problem. If not, ask them. If you do find that it comes back to some problem with Material for MkDocs then please do reach out again.
